docs/branches.md

```diff
@@ -138,6 +138,17 @@ Branches that already existed before the `jj git fetch` are not affected. This
 is similar to Mercurial, which fetches all its bookmarks (equivalent to Git
 branches) by default.
 
+## Branch movement
+
+Currently Jujutsu only moves branches by itself when these conditions are met:
```

Comment:
nit: `s/branches/local branches/` for clarity?

Reply:
done